src/components/Albums.jsx

```diff
@@ -0,0 +1,36 @@
+  },
+  {
+    albumId: 2,
+    name: 'folklore',
+    artist: 'Taylor Swift',
+    cover: 'https://m.media-amazon.com/images/I/81OReT+-6JL._SL1200_.jpg',
+  },
+  {
+    albumId: 3,
+    name: 'Donda',
+    artist: 'Kanye West',
+    cover: 'https://www.esonosoy.com/wp-content/uploads/2021/09/donda-kanye-west.jpg',
+  }];
+
+function Albums() {  
+  return (
+    <div>
+      <ul class="albums">
```

Comment:
Un pequeño detalle respecto al uso de clases en componentes React:
```suggestion
      <ul className="albums">
```